### PR TITLE
Optimise`at-sign` icon

### PR DESCRIPTION
**Pull request by danielbayley**

### CODE REVIEW

**Comment by karsa-mistmere** [comment]
Can you provide an explanation as to why this change would be beneficial?

**Comment by danielbayley** [comment]
> Can you provide an explanation as to why this change would be beneficial?

Well it started as a simple optimisation…
~~~ svg
<path d="M16 8v5a3 3 0 0 0 6 0v-1a10 10 0 1 0-4 8" />
~~~
vs the existing
~~~ svg
<path d="M16 8v5a3 3 0 0 0 6 0v-1a10 10 0 1 0-3.92 7.94" />
~~~

But also I prefer without the ascender, like octicons (https://primer.style/design/foundations/icons/mention-16)… Just me then?

Now separate `commit`s, so either way…

**Comment by ericfennis** [comment]
Not sure about this change, I think the existing is more recognizable.

**Comment by danielbayley** [comment]
> Closing this, since neither @ericfennis, @jguddas or are seem to be in favour.

Right, but as mentioned…

> Well it started as a simple optimisation…
> ~~~ svg
> <path d="M16 8v5a3 3 0 0 0 6 0v-1a10 10 0 1 0-4 8" />
> ~~~
> vs the existing
> ~~~ svg
> <path d="M16 8v5a3 3 0 0 0 6 0v-1a10 10 0 1 0-3.92 7.94" />
> ~~~

> Now separate `commit`s, so either way…

@karsa-mistmere I can’t revert the last commit (restoring the ascender, but leaving the optimisation) since you closed this, and seems silly to bother opening another PR…

File: icons/at-sign.svg [hunk 1]
```diff
@@ -10,5 +10,5 @@
   stroke-linejoin="round"
 >
   <circle cx="12" cy="12" r="4" />
-  <path d="M16 8v5a3 3 0 0 0 6 0v-1a10 10 0 1 0-3.92 7.94" />
+  <path d="M18 20a10 10 0 1 1 4-8v1a3 3 0 1 1-6 0v-1" />
```
Comment (karsa-mistmere) [inline icons/at-sign.svg]:
```suggestion
  <path d="M18 20a10 10 0 1 1 4-8v1a3 3 0 1 1-6 0V8" />
```

**Comment by karsa-mistmere** [comment]
> @karsa-mistmere I can’t revert the last commit (restoring the ascender, but leaving the optimisation) since you closed this, and seems silly to bother opening another PR…

Right, sorry, reopened.

**Comment by jguddas** [comment]
I think you should also be able to push to branches with closed PRs.

**Pull request merged**